tests/Unit/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.py

```diff
@@ -2,26 +2,16 @@
 # See LICENSE.txt for details.
 
 import numpy as np
-import os
-import sys
+
 from PointwiseFunctions.GeneralRelativity.ComputeGhQuantities import (
     spacetime_deriv_detg, deriv_lapse, dt_lapse, deriv_spatial_metric,
     dt_spatial_metric, deriv_shift, dt_shift)
 from PointwiseFunctions.GeneralRelativity.ComputeSpacetimeQuantities import (
     derivatives_of_spacetime_metric, inverse_spacetime_metric,
     spacetime_normal_vector)
-
-
-def spatial_weight_function(coords, r_max):
-    r2 = np.sum([coords[i]**2 for i in range(len(coords))])
-    return np.exp(-r2 / r_max / r_max)
-
-
-def spacetime_deriv_spatial_weight_function(coords, r_max):
-    W = spatial_weight_function(coords, r_max)
-    DW = np.zeros(len(coords) + 1)
-    DW[1:] = -2. * W / r_max / r_max * coords
-    return DW
+from .DampedWaveHelpers import (spatial_weight_function,
```

Comment:
Why the . before DampedWaveHelpers here?

Reply:
That tells python to load the file relative to the current file's directory rather than check the python source paths. Going from the python source paths would make this line something like 120 chars long